data/summary/types/summary.go

```diff
@@ -72,7 +79,8 @@ type Config struct {
 }
 
 type Dates struct {
-	LastUpdatedDate time.Time `json:"lastUpdatedDate" bson:"lastUpdatedDate"`
+	LastUpdatedDate   time.Time `json:"lastUpdatedDate" bson:"lastUpdatedDate"`
```

Comment:
For my understanding, is this always populated?

Reply:
yes, always. its created with a 0 value on backfill to push backfills to the front of the queue, but otherwise its always filled with real values, any Zero default values are intentional